src/test/java/com/meilisearch/integration/SearchNestedTest.java

```diff
@@ -0,0 +1,125 @@
+package com.meilisearch.integration;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.meilisearch.integration.classes.AbstractIT;
+import com.meilisearch.integration.classes.TestData;
+import com.meilisearch.sdk.Index;
+import com.meilisearch.sdk.SearchRequest;
+import com.meilisearch.sdk.Settings;
+import com.meilisearch.sdk.Task;
+import com.meilisearch.sdk.json.GsonJsonHandler;
+import com.meilisearch.sdk.utils.Movie;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("integration")
+public class SearchNestedTest extends AbstractIT {
+
+    private TestData<Movie> testData;
+
+    final class Results {
```

Comment:
This class already exist in other places right?

Reply:
Yes, in the `SearchTest` file.